app/src/main/java/com/keylesspalace/tusky/settings/SettingsConstants.kt

```diff
@@ -45,7 +45,7 @@ enum class AppTheme(val value: String) {
  *
  * - Adding a new preference that does not change the interpretation of an existing preference
  */
-const val SCHEMA_VERSION = 2023082301
+const val SCHEMA_VERSION = 2023112001
```

Comment:
I don't know if this way of updating SharedPreferences is correct.

So, please point out if I am wrong!

Reply:
You did it correctly